src/Microsoft.ML.Data/Transforms/Hashing.cs

```diff
@@ -1132,49 +1132,49 @@ public sealed class ColumnOptions
             /// <summary> Name of column to transform.</summary>
             public readonly string InputColumnName;
             /// <summary> Number of bits to hash into. Must be between 1 and 31, inclusive.</summary>
-            public readonly int HashBits;
+            public readonly int NumberOfHashBits;
             /// <summary> Hashing seed.</summary>
             public readonly uint Seed;
             /// <summary> Whether the position of each term should be included in the hash.</summary>
-            public readonly bool Ordered;
+            public readonly bool UseOrderedHashing;
```

Comment:
>UseOrderedHashing [](start = 33, length = 17)

I don't like this one as well.
First: it's again make sense only for vectors.
What it does:
you have vector "A","A" in first row and "A","B" in second and "B","A" in third.
"A" maps to 1, "B" maps to 2 with hash function we use (let's assume that for simplicity)

by default it would map that three rows to 1,1; 1,2;  2,1
But if you turn it on it will map it to 1,3; 1,4; 2;3
3 is from hash("A"+1)
4 is from hash("B"+1)

Do I have better suggestion? No....

Reply:
I actually don't mind this name, especially with the documentation that goes with it. Hashing is not something that is order dependent usually. While here it is.

---
In reply to: [263639956](https://github.com/dotnet/machinelearning/pull/2862#discussion_r263639956) [](ancestors = 263639956)

Reply:
I have added a few words saying it only applies to inputs of vector type.

---
In reply to: [263972348](https://github.com/dotnet/machinelearning/pull/2862#discussion_r263972348) [](ancestors = 263972348,263639956)